### ISSUE TITLE
The variable has not been defined in the model.

### ISSUE THREAD

**Issue by ClongPang** [fqldom/BeFA#2]
def save_best(self):
        torch.save(self.gate_v.state_dict(),
                   os.path.join(self.save_path, 'gate_v_{}_{}.pt'.format(self.model, self.dataset)))
        torch.save(self.gate_t.state_dict(),
                   os.path.join(self.save_path, 'gate_t_{}_{}.pt'.format(self.model, self.dataset)))
        torch.save(self.image_trs.state_dict(),
                   os.path.join(self.save_path, 'image_trs_{}_{}.pt'.format(self.model, self.dataset)))
        torch.save(self.text_trs.state_dict(),
                   os.path.join(self.save_path, 'text_trs_{}_{}.pt'.format(self.model, self.dataset)))
        np.save(os.path.join(self.save_path, 'item_id_embedding_{}_{}'.format(self.model, self.dataset)),
                self.item_id_embedding.weight.detach().cpu().numpy())

self.gate_v和self.gate_t has not been defined in the model.
